cpp/oneapi/dal/data/graph_service_functions.hpp

```diff
@@ -15,12 +15,6 @@
  * limitations under the License.
  *******************************************************************************/
 
-/*
-//++
-//  Graph types and service functionality
-//--
-*/
```

Comment:
why was it removed? what is the style ?

Reply:
Documentation will be updated in separate pull request. The style removed here was incorrect